### DISCUSSION TITLE
How to plot vertically (Vsv) and horizontally (Vsh) polarized shear wave vertically

### DISCUSSION THREAD

**Discussion by bjzhou96** [mtex-toolbox/mtex#1380] [Ask Anything]
Hello everyone

I am doing some research on mineral deformation and seismic anisotropy. I have already plot the distribution of P wave, S1 and S2 based on CPO data by MTEX. Now I am wondering if and how MTEX can plot the azimuthal variations in vertically and horizontally polarized shear waves(Sv and Sh), like (c) in the figure?

I wil appeciate it if anyone can help me. Thank you in advance.

Baojun Zhou
![image](https://user-images.githubusercontent.com/58097480/158169035-acfe3f0a-1c69-4422-87aa-28871c35a25c.png)

**Comment by kilir** [comment]
Hi,
do you mean something like that?
```
C = Forsterite1989 % example input
odf = unimodalODF(orientation.rand(1,C.CS),'haltwidth',20*degree)

[CVoigt_ol, CReuss_ol, CHill_ol]    = mean(C,odf);

[vp,vs1,vs2,pp,ps1,ps2] = velocity(CReuss_ol,'harmonic')

% "horizontal reference directions"
ap = vector3d.byPolar(90*degree,[-90:90]*degree,'antipodal')

plot(ap,eval(vs1,ap))

nextAxis

plot(ap,eval(vs2,ap))

```

Cheers,
Rüdiger

**Reply by bjzhou96** [reply]
Hello Rüdiger

Thank you for your reply! Sorry for my confusing expression. I tried your code, and I am grateful that you already know what I mean. Here is two questions. (1) The variations of vs1 and vs2 is not what I want. What I want to calculate is the variations of vertically polarized VS and horizontally polarized VS, that is VSv and VSh. It is something like vertical projection and horizontal projection of VS. And it is not about shearing wave splitting vs1 or vs2. I don't know if there are these kind of variables in MTEX, or can we calculate them by some function? (2) Can we plot these variations in the form of lines like (c) in the figure, instead of on a circle ?

Best wishes
Baojun Zhou

**Reply by kilir** [reply]
Hi,
wrt (1), can you give a precise description of what you are looking for in that case? 
wrt(2), simply plot values as a function of the angles.
Cheers,
Rüdiger

**Reply by bjzhou96** [reply]
Hello Rüdiger

I want to calculate the distributin of velocity of SH and SV, like the plot in the following picture. I don't know if MTEX defines these two variables. They are different from V1 and V2.
Best wishes
Baojun Zhou

![image](https://user-images.githubusercontent.com/58097480/158598174-d9f3d34c-7a02-4019-ad2d-098074409517.png)

**Reply by kilir** [reply]
Hi,
maybe the reference of your figure might bring some more information.
Rüdiger

**Reply by bjzhou96** [reply]
Hello Rüdiger

I find the definition for Vsh and Vsv on https://mtex-toolbox.github.io/SeismicVelocitySingleCrystalDemo2d.html by Professor David Mainprice. will it be helpful for expressing what I mean? 

Best wishes,
Baojun Zhou

![image](https://user-images.githubusercontent.com/58097480/158619502-368376ab-7302-4bc5-ac41-49cf4d195b69.png)

**Reply by kilir** [reply]
Even better. The examples are in Mtex, so you can use them directly.
Cheers,
Rüdiger
